src/foam/box/SessionServerBox.java

```diff
@@ -177,6 +178,12 @@ public void send(Message msg) {
         return;
       }
 
+      // Sub context might have service override for the delegate
+      if ( effectiveContext instanceof SubX ) {
```

Comment:
@lchanmann  Can you elaborate on this, for my benefit, Thanks.

Reply:
I added "bepay.brazilVerificationService" nspec but the original "brazilVerificationService" which is a box/skeleton service was created earlier so we need to override the delegate of the skeleton to point to the bepay specific brazilVerificationService.